infra/gcp/ensure-main-project.sh

```diff
@@ -167,6 +173,48 @@ function ensure_terraform_state_buckets() {
     done
 }
 
+function ensure_prow_buckets() {
+    if [ $# -ne 1 ] || [ -z "$1" ]; then
+        echo "${FUNCNAME[0]}(gcp_project) requires 1 argument" >&2
+        return 1
+    fi
+
+    local project="${1}"
+
+    for bucket in "${PROW_BUCKETS[@]}"; do
+        local svc_acct_name="${bucket}-sa"
```

Comment:
I think you are going to want to reuse the existing service account(s) that prow jobs run as instead of creating a new service account

Reply:
So tl;dr I would create a `k8s-infra-prow@kubernetes-public` service account for the prow control plane, since it may require more than just write access to this one bucket.

Since you're doing this for the cluster in kubernetes-public, you could do all this in https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/clusters/projects/kubernetes-public/aaa

But no objections to iterating in bash if you need to.

Reply:
@spiffxp I'll add a second commit with a HCL version.

Reply:
@spiffxp The Terraform version is done in [`03d55ef`](https://github.com/kubernetes/k8s.io/pull/1909/commits/03d55efa685fda4718553202c171b378029223dc).